tests/foreman/api/test_capsule.py

```diff
@@ -0,0 +1,227 @@
+"""Tests for the ``smart_proxies`` paths.
+
+:Requirement: Smartproxy
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Capsule
+
+:Team: Endeavour
+
+:TestType: Functional
+
+:CaseImportance: Critical
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+from fauxfactory import gen_url
+from requests import HTTPError
+
+from robottelo.config import user_nailgun_config
+
+
+pytestmark = [pytest.mark.run_in_one_thread]
```

Comment:
Probably this is not in use, and anyways all component tests runs serial on xdist worker. So this wont be needed.